server/index.js

```diff
@@ -28,13 +28,16 @@ const createNomisPushService = require('./services/nomisPushService')
 const createDeadlineService = require('./services/deadlineService')
 const createJobSchedulerService = require('./services/jobSchedulerService')
 const createNotificationJobs = require('./services/jobs/notificationJobs')
+const createNomisRoService = require('./services/nomisRoService')
 
 const signInService = createSignInService(audit)
 const licenceService = createLicenceService(licenceClient)
 const conditionsService = createConditionsService(config)
-const prisonerService = createPrisonerService(nomisClientBuilder)
+const nomisRoService = createNomisRoService(nomisClientBuilder)
+const roService = nomisRoService
```

Comment:
I realise this looks redundant but in the next PR this is where the feature switching will happen
eg
`const roService = config.roServiceType === 'DELIUS' ? deliusRoService : nomisRoService`